Remotion/Web/Core/UI/Controls/FormGridLabel.cs

```diff
@@ -37,9 +37,37 @@ namespace Remotion.Web.UI.Controls
 [ToolboxItemFilter("System.Web.UI")]
 public class FormGridLabel: Label, ISmartControl
 {
+  private const string c_textViewStateKey = nameof (Text);
+  private const string c_textWebStringViewStateKey = c_textViewStateKey + "_" + nameof (WebStringType);
+
   private bool _required = false;
   private string? _helpUrl = null;
 
+  [Category("Appearance")]
+  [Description("Label_Text")]
+  [DefaultValue("")]
```

Comment:
Please resolve the Label_Text string
Please check if we should add typeof (WebString) next to the empty string.